### ISSUE TITLE
Upgrade to 3.9.3

### ISSUE THREAD

**Issue by isbn390** [emissary-ingress/emissary#5512]
We were having filters and filterpolicy on our gke cluster, after the update of ambassador from 3.7 to 3.9.3, filter and filterpolicy is missing. kubectl get is returning no resource found. But both the resource is there in the cluster, since our app is working fine and without these resources we will not be able to login to the app. 

We try to reapply these using the yaml file we had and we are getting 
filter.getambassador.io/test-filter unchanged.

So we try to run kubectl get filter test-filter -o yaml to get the yaml of the filter and we are getting
Error from server (NotFound): filters.gateway.getambassador.io "test-filter" not found

Bottom line is, In 3.9.3 kubectl get filter is not returing the filters present

**Update**: My resources are in version v2, I think I need to convert it to v3alpha1. Can anyone guide me to how to do so?
